IBAnimatable/AnimatablePresentationController.swift

```diff
@@ -45,9 +50,12 @@ public class AnimatablePresentationController: UIPresentationController {
 
 private extension AnimatablePresentationController {
 
-  func setupDimmingView() {
-    let tap = UITapGestureRecognizer(target: self, action: #selector(chromeViewTapped))
+  func configureDimmingView() {
+    var tap = UITapGestureRecognizer(target: self, action: #selector(dimmingViewTapped))
     dimmingView.addGestureRecognizer(tap)
+    tap = UITapGestureRecognizer(target: self, action: #selector(dimmingViewTapped))
```

Comment:
Not sure, since they are exactly the same and we don't stored them as a property. I can change it if you think that's better